### DISCUSSION TITLE
Detailed steps

### DISCUSSION THREAD

**Discussion by rjhuzy** [velsa/notehost#35] [General]
Hi All,

New to Github, but I somewhat understand the code and Fruition has detailed steps on how to deploy things.
Since that is out of maintenance, I looking forward to NoteHost as a solution.

So yeah can someone please share detailed steps or a YT video for guidance as the same Fruition has Dimitris Solutions video

**Comment by slau8** [comment]
A little late, but instructions were posted [here](https://github.com/velsa/notehost/discussions/26). Worked for me.
